clusterloader2/pkg/measurement/common/slos/pod_startup_latency.go

```diff
@@ -113,6 +114,11 @@ func (p *podStartupLatencyMeasurement) start(c clientset.Interface) error {
 	klog.Infof("%s: starting pod startup latency measurement...", p)
 	p.isRunning = true
 	p.stopCh = make(chan struct{})
+	wait.Until(func() {
+		if err := p.gatherScheduleTimes(c); err != nil {
```

Comment:
What's the point of listing events immediately on startup - the list is expensive and will not given any results anyway.

Reply:
Fixed.